### ISSUE TITLE
A11y_AzureToolsStorageExplorer_Application_ScreenReader: NVDA not announces status message appearing in "Activities" tab when "Use device code flow sign-in(preview)" check box is checked.  

### ISSUE THREAD

**Issue by Suraj1200499** [microsoft/AzureStorageExplorer#1926]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags: #A11y_AzureToolsStorageExplorer; #A11yWCAG2.1; #A11YTCS; #DesktopApp; #MAS4.1.3; #MAS1.3.1; #NVDA; #Win10;

**Environment Details:**
Application Name: Azure Tools Storage Explorer
Application Version: 1.10.0
Windows10

**Repro Steps:**
1. Launch Storage Explorer
2. Navigate to Setting menu on left pane and click on it.
3. Navigate to Application on right pane and  click on it.
4. "Application" screen gets opened.
5. Verify that all the elements of "Application" Screen are accessible and MAS Compliant.
6. Start NVDA.
7. Navigate using Tab key to check box "Use device code flow sign-in(preview)" and check the check box.
8. Verify after checking below in Activities tab message appears but that status message NVDA not announces.

**​Actual:​**
When check box "Use device code flow sign-in(preview)"  is checked, below in "Activities" tab status message(Preview feature "Use Device Code Flow Sign-in" is enabled) appears but this message NVDA not announces.
And when check box is unchecked status message(Preview feature "Use Device Code Flow Sign-in" is disabled) appears this message also NVDA not announces.

Same issue is observed in "**JAWS**" also.

**Same issue is observed in below Scenarios:**
1. **Open button:**
* Here in this Scenario NVDA not announces the Status message which is appearing in "Activities" tab after hitting enter on "Open" control.
2. **Transfers:**
* Here in this Scenario NVDA not announces status message which is appearing Activities tab when "Use AZCopy for improved blob upload and download(preview)" is checked. 

**Expected:**
When check box "Use device code flow sign-in(preview)" is checked/unchecked, NVDA should announce Status message which appears in "Activities" tab. 

**UserImpact:**
Screen reader users gets may not be able to know about the status message if screen reader not announces the status message which appears in "Activities" tab.

**Recommendation:**
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/Doc.aspx?sourcedoc=%7B684573A7-B089-4131-9B39-0009054125D3%7D&file=MAS%204.1.3%20%E2%80%93%20Status%20Messages.docx&action=default&mobileredirect=true&cid=62f1e149-3e64-42c6-b14f-39da49914fd3

https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment for Reference:**
[A11y_AzureToolsStorageExplorer_Application_ScreenReader.pptx](https://github.com/microsoft/AzureStorageExplorer/files/3657253/A11y_AzureToolsStorageExplorer_Application_ScreenReader.pptx)

**Comment by MRayermannMSFT** [comment]
This UI (popping up the info message when enabling device code flow) does not exist anymore in our latest version (1.11.0). It will not return.

**Comment by MRayermannMSFT** [comment]
@Radha2019 @Charanrajkb @nirmalsahoo @Suraj1200499 @vinodbk197

Please verify whether or not this bug was fixed to your satisfaction in the latest release of Storage Explorer. You can obtain that release from https://www.storageexplorer.com

If the bug has not been fixed to your satisfaction, please:
- Explain how did the fix not meet your expectations.
- Tell us exactly how we can meet your expectations. If there is more than one way to do so, feel free to list the different ways.

**Comment by vinodbk197** [comment]
Verified this issue on:Azure Tools Storage Explorer (Version: 1.12.0)
The issue is no longer to reproduces, Hence closing this bug.

**Comment by fsteffi** [comment]
GithubTags:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-StorageExplorer-Win32-Sep2019;A11yMAS;

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-StorageExplorer-Win32-Sep2019;
